src/contourFeature.js

```diff
@@ -63,7 +64,8 @@ var meshFeature = require('./meshFeature');
  * @property {number[]} [rangeValues] An array used to map values to the
  *    `colorRange`.  By default, values are spaced linearly.  If specified, the
  *    entries must be increasing weakly monotonic, and there must be one more
- *    entry then the length of `colorRange`.
+ *    entry then the length of `colorRange` if the contour is stepped, or the
```

Comment:
stepped or unstepped may not be obvious to some, should we also add "continuous" and "discreet"

Reply:
Should we just change the documentation, or do you think we should allow multiple names for the parameter, too?

Reply:
for the backward compatibility, adding multiple names for the parameters makes sense to me.

Reply:
Okay -- let's do that in a separate PR.

Reply:
sounds good